package.json

```diff
@@ -124,6 +124,7 @@
     "jest-junit": "^12.0.0",
     "lint-staged": "^10.1.3",
     "prettier": "2.1.2",
+    "react-is": "^16.13.1",
```

Comment:
what u used to determine this version?

Reply:
It's a sub-package in the `react` repo, and as such the versions almost perfectly align with `react` release. There's no `16.14`, though, and so I thought `^17.x` wasn't appropriate since we're not yet on it.